src/ToolWindowManager.cpp

```diff
@@ -151,8 +151,8 @@ void ToolWindowManager::addToolWindows(QList<QWidget *> toolWindows, const ToolW
     toolWindow->hide();
     toolWindow->setParent(0);
     m_toolWindows << toolWindow;
-    m_toolWindowProperties[toolWindow] = ToolWindowProperty(0);
-    QObject::connect(toolWindow, &QWidget::windowTitleChanged, this, &ToolWindowManager::windowTitleChanged);
+    m_toolWindowProperties[toolWindow] = properties;
+    QObject::connect(toolWindow, &QWidget::windowTitleChanged, this, &ToolWindowManager::windowTitleChanged);    
```

Comment:
There's some added trailing whitespace here.

Reply:
Removed it in next commit.